src/components/FoodNutrientsComponent/FoodNutrientsComponent.jsx

```diff
@@ -1,17 +1,20 @@
 import React, { Component } from "react";
```

Comment:
this component will need a back button to go back to the nutrition view.  It should also reset upon clicking the addFoodComponent, because right now a food submitted for breakfast will still display if the user goes to lunch.

can be added to the debug list.